packages/arb-token-bridge-ui/src/components/common/WithdrawalCountdown.tsx

```diff
@@ -15,28 +16,28 @@ export function WithdrawalCountdown({
   } = useNetworksAndSigners()
   const { currentL1BlockNumber } = useAppContextState()
 
-  const blockTime = l1Network?.blockTime || 15
-
   if (
     typeof nodeBlockDeadline === 'undefined' ||
     typeof l2Network === 'undefined'
   ) {
     return <span>Calculating...</span>
   }
 
+  const blockTime = getBlockTime(l1Network.chainID) || 15
+  const confirmPeriodBlocks = getConfirmPeriodBlocks(l2Network.chainID)
+
+  if (nodeBlockDeadline === 'EXECUTE_CALL_EXCEPTION' || !confirmPeriodBlocks) {
+    return <span>Failure</span>
```

Comment:
should we show Failure in the case of undefined confirmPeriodBlocks?

Reply:
I think we should just let it throw in this case and surface the error in Sentry, as this should not happen ever when using the right networks.